widgetssdk/src/main/java/com/glia/widgets/permissions/PermissionsSupportActivity.kt

```diff
@@ -0,0 +1,5 @@
+package com.glia.widgets.permissions
+
+import androidx.appcompat.app.AppCompatActivity
+
+class PermissionsSupportActivity : AppCompatActivity()
```

Comment:
NIT: Even though everyone who has worked with permission can figure out why this class is needed based on its name but I think it would be nice to have some comments in here for the future.